docs/install.md

```diff
@@ -8,15 +8,56 @@ The MarkLogic Koop Provider can be used in one of two ways - either via a downlo
 ready-to-run application that can be configured for your environment, or via a new Koop application created with the
 Koop CLI.
 
-Both of these approaches require that you have a MarkLogic application with the
-[MarkLogic Geo Data Services (GDS)](https://github.com/marklogic-community/marklogic-geo-data-services) modules
-installed. Please see the GDS documentation for adding these modules to your MarkLogic application.
+Both of these approaches require that you have a MarkLogic application with
+[MarkLogic Geo Data Services (GDS)](https://github.com/marklogic-community/marklogic-geo-data-services) installed. 
+Please see 
+[the GDS documentation](https://marklogic-community.github.io/marklogic-geo-data-services/) for instructions on 
+adding GDS to your MarkLogic application. 
 
-### Using the application zip
+## Using the application zip
```

Comment:
This is the main new stuff for when a user just wants to download a zip to get going and doesn't want to use Koop's quickstart.